ts/src/checker.ts

```diff
@@ -31,6 +30,24 @@ const fsAccess = promisify(fs.access);
 const fsReadDir = promisify(fs.readdir);
 const fsReadFile = promisify(fs.readFile);
 
+// Valid license IDs defined in https://spdx.org/licenses/ must be used whenever
+// possible. When adding new licenses, please consult the relevant documents and
+// OSPO.
+const DEFAULT_GREEN_LICENSES = [
```

Comment:
This list should probably be documented in the README.

Reply:
Yup. Done.